LDMP/jobs/mvc.py

```diff
@@ -265,29 +305,91 @@ def __init__(self, job: models.Job, parent=None):
-        job_directory = manager.job_manager.get_job_file_path(self.job).parent
-        # NOTE: not using QDesktopServices.openUrl here, since it seems to not be
-        # working correctly (as of Jun 2021 on Ubuntu)
-        openFolder(str(job_directory))
 
-    def load_dataset(self):
-        manager.job_manager.display_job_results(self.job)
+class DatasetDetailsWidget(QtWidgets.QDialog, DatasetActions, WidgetDatasetItemDetailsUi):
 
-    def delete_dataset(self):
-        message_box = QtWidgets.QMessageBox()
-        message_box.setText(
-            f"You are about to delete job {self.job.params.task_name!r}")
-        message_box.setInformativeText("Confirm?")
-        message_box.setStandardButtons(
-            QtWidgets.QMessageBox.Yes | QtWidgets.QMessageBox.Cancel)
-        message_box.setDefaultButton(QtWidgets.QMessageBox.Cancel)
-        message_box.setIcon(QtWidgets.QMessageBox.Information)
-        result = QtWidgets.QMessageBox.Res = message_box.exec_()
-        if result == QtWidgets.QMessageBox.Yes:
-            log("About to delete the dataset")
-            for path in self.job.results.local_paths:
-                layers.delete_layer_by_filename(str(path))
-            manager.job_manager.delete_job(self.job)
\ No newline at end of file
+    def __init__(self, job: models.Job, parent=None):
+        super(DatasetDetailsWidget, self).__init__(parent)
+        self.setupUi(self)
+
+        self.job = job
+
+        self.name_le.setText(self.job.params.task_name)
+        self.state_le.setText(self.job.status.value)
+        self.created_at_le.setText(str(self.job.start_date))
+
+        self.load_btn.clicked.connect(self.load_dataset)
+        self.delete_btn.clicked.connect(self.__delete_dataset)
+        self.open_directory_btn.clicked.connect(self.open_job_directory)
+        self.export_btn.clicked.connect(self.export_dataset)
+        self.delete_btn.setEnabled(
+            self.job.status == models.JobStatus.DOWNLOADED
+        )
+
+        self.alg_le.setText(self.job.script.name)
+        self.path = None
+
+        for path in self.job.results.local_paths:
+            if 'tif' in str(path):
+                self.path = str(path)
+        self.path_le.setText(self.path)
+
+        self.load_btn.setIcon(
+            QtGui.QIcon(':/plugins/LDMP/icons/mActionAddRasterLayer.svg'))
+        self.open_directory_btn.setIcon(
+            QtGui.QIcon(':/images/themes/default/mActionFileOpen.svg'))
+        self.export_btn.setIcon(
+            QtGui.QIcon(':/plugins/LDMP/icons/export_zip.svg'))
+        self.delete_btn.setIcon(
+            QtGui.QIcon(':/plugins/LDMP/icons/mActionDeleteSelected.svg'))
+
+        self.load_job_details()
+
+        self.bar = qgis.gui.QgsMessageBar()
+        self.bar.setSizePolicy(
+            QtWidgets.QSizePolicy.Minimum, QtWidgets.QSizePolicy.Fixed
+        )
+        self.layout().addWidget(self.bar, 0, 0, alignment=QtCore.Qt.AlignTop)
+
+    def __delete_dataset(self):
```

Comment:
Please do not use this kind of naming scheme for methods. Don't use double underscores for naming methods that are not Python special methods. I am aware some devs do it to signal that this is a private method. The usefulness of this in a language like Python can be debated, but for brevity, for this project, lets just no use it.